src/transaction.ts

```diff
@@ -1477,13 +1490,15 @@ export class Transaction extends Dml {
    * });
    */
   commit(
-    gaxOptionsOrCallback?: CallOptions | CommitCallback,
+    optionsOrCallback?: CommitOptions | CommitCallback,
```

Comment:
@AVaksman I would like your input here. I would like to be able to make this change without it being a breaking change. Is there a way to pass in two kinds of options? E.g. commitOptions and gaxOptions.

Reply:
I think I sorted it out based on a suggestion by @olavloite.